Dockerfile.template.erb

```diff
@@ -17,7 +17,7 @@ FROM mcr.microsoft.com/windows/servercore:<%= windows_ver %>
 <% if is_armhf %>
 FROM resin/armhf-alpine:3.7
 <% else %>
-FROM alpine:3.12
+FROM alpine:3.13.5
```

Comment:
From DockerHub, `alpine:3.13` and `alpine:3.13.5` have same sha256 hash:
https://hub.docker.com/_/alpine?tab=tags&page=1&ordering=last_updated&name=3.13

For maintainability, I think that we should use `alpine:3.13` instead of `alpine:3.13.5`.

Reply:
Thanks! it should be alpine:3.13 surely.